gusto/timeloop.py

```diff
@@ -125,5 +147,50 @@ def run(self, t, tmax, pickup=False):
                 state.dump(t, pickup=False)
 
         state.diagnostic_dump()
-
         print "TIMELOOP complete. t= "+str(t-dt)+" tmax="+str(tmax)
```

Comment:
In parallel, these kind of prints are pretty unfriendly.  If you do:

```python
from firedrake.petsc import PETSc

PETSc.Sys.Print("some string", comm=self.state.mesh.comm)
```

Then you'll only print on rank zero of the provided communicator.